fuzzers/afl/fuzzer.py

```diff
@@ -124,6 +124,7 @@ def run_afl_fuzz(input_corpus,
         # performs.
         '2147483647'
     ]
+    os.environ['AFL_SHUFFLE_QUEUE'] = '1'
```

Comment:
Please move this into prepare_fuzz_environment

Reply:
done